core/src/main/java/dk/alexandra/fresco/lib/helper/bristol/BristolCircuitParser.java

```diff
@@ -111,8 +107,7 @@ private void parseLine(String line, ProtocolBuilderBinary builder) throws IOExce
     // System.out.println("IN: " + Arrays.toString(in));
     // System.out.println("OUT: " + Arrays.toString(out));
 
-    switch (type) {
```

Comment:
Switch stmt makes it harder to do 100% testing? I see no other reason for changing this.

Reply:
I don't really see the reason either. At least it should be `if ... else ...` i guess. Also, please delete all the commented code.

Reply:
... also you could reduce the complexity of this code by having just a single return statement at the end. There is really no reason to have them in each case.

Reply:
The switch is hard to test (apparently it leaves out 3 branches somehow), but I have added else statements.